### PR TITLE
reorganize recipe card customization doc

### PR DESCRIPTION
**Pull request by ThibaultMiam**

### CODE REVIEW

**Comment by ThibaultMiam** [comment]
* **#77** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/77?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ThibaultMiam and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+label: Recipe card customization
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import UseStringResource from '../../../_code/_utils/UseStringResource.mdx'
+
+# RecipeCard customization
+
+RecipeCard is one of the most use components, it has three customization states: **success**, **loading** & **empty**
```
Comment (antonin-francois) [inline docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx:12]:
```suggestion
RecipeCard is one of the most used components, it has three customization states: **success**, **loading** & **empty**
```

File: docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx [hunk 2]
```diff
@@ -0,0 +1,770 @@
...
+                view = RecipeCardShelfLoadingCustomView()
+            }
+        }
+    }
+}
+```
+
+## Success
+
+Recipe card success is one of those three states, it's visible when all data has been fetched without any error.
```
Comment (antonin-francois) [inline docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx:48]:
```suggestion
Recipe card success is one of those three states, it's visible when all of the data has been fetched without any error. 
```

File: docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx [hunk 3]
```diff
@@ -0,0 +1,770 @@
...
+            }
+        }
+    }
+}
+```
+
+## Success
+
+Recipe card success is one of those three states, it's visible when all data has been fetched without any error. 
+To create your own recipe card success template you create a class that implements `RecipeCardSuccess`
```
Comment (antonin-francois) [inline docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx:49]:
```suggestion
To create your own recipe card success template you create a class that implements `RecipeCardSuccess`.
```

File: docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx [hunk 4]
```diff
@@ -0,0 +1,770 @@
...
+
+:::warning
+View temmplate is **Deprecated** you must use **catalog** and **shelf** instead
```
Comment (antonin-francois) [inline docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx:52]:
```suggestion
View template is **Deprecated** you must use **catalog** and **shelf** instead
```

File: docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx [hunk 5]
```diff
@@ -0,0 +1,770 @@
...
+
+| Name | Ressource ID | Value Fr | Value Eng |
+| - | - | - | - |
+| showDetails | com_miam_recipe_is_in_cart | Voir le détail | Show details |
+| add | com_miam_recipe_add | Ajouter les ingrédients | Add ingredients |
+| perPerson | com_miam_my_meals_per_person | /pers.| /pers. |
+
+## RecipeCard Loading
+
+As for Success state basic Laoding view has been **deprecated** you must use catalog and shelf loading.
```
Comment (antonin-francois) [inline docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx:648]:
```suggestion
As for the Success state, the basic Loading view has been **deprecated** you must use catalog and shelf loading.
```

File: docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx [hunk 6]
```diff
@@ -0,0 +1,770 @@
...
+        }
+    }
+}
+```
+</TabItem>
+</Tabs>
+
+###  Loading catalog
+
+You can choose to have to differents design for the recipe card in our catalog and in your shelf.
```
Comment (antonin-francois) [inline docs/android/features/recipe-card/customize-view/RecipeCard-customization.mdx:689]:
```suggestion
You can choose to have to different designs for the recipe card in our catalog and in your shelf.
```

**Review by antonin-francois — CHANGES_REQUESTED** [review]

**Review by antonin-francois — APPROVED** [review]

**Review by diarmuidmcg — APPROVED** [review]

**Review by antonin-francois — CHANGES_REQUESTED** [review]
c'est 4.0 ou 4.1 ou les deux ?

**Review by antonin-francois — APPROVED** [review]

**Review by diarmuidmcg — APPROVED** [review]

**Pull request merged**